### PR TITLE
Update Abstract

### PR DESCRIPTION
**Pull request by tshemsedinov**
- [x] check spelling
- [x] execute `npm run format` before open PR

### CODE REVIEW

**Comment by tshemsedinov** [comment]
@tursky Please review a small fix in abstract

**Comment by tursky** [comment]
> @tursky Please review a small fix in abstract

The Czech language has a Slavic type, but in structure it is closer to Latin. For Czechs, the location of a word in a sentence sometimes matters, as well as a certain musicality of speech. The new version of the translation is not correct, but intuitively it will be understandable.

Your speech construction in this case is composed in a functional paradigm, which complicates its translation. Czechs don't talk like that. And I personally did not meet the word "осмысление". It has a philosophical meaning and will overflow the reader's short-term memory stack.

But after some searching, I can offer a relatively close to Russian translation in a functional style, adapted for the Czech listener:

- Pokusíme se zvážit techniky a principy různých programovacích paradigmat prizmatem metaprogramování a změnit jestli ne samotné softwarové inženýrství tím, tak aspoň rozšířit pochopení disciplíny novými generacemi profesionálů.

File: content/Abstract.cz.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
...
... jak vytvořit metadata, která dávají pravidla a posloupnost toku zpracování jiných dat. Multiparadigmatické programování bere to nejlepší ze všech paradigmat a sestavuje z něj syntaktické konstrukce, které umožňují popsat předmětnou oblast jasněji a pohodlněji. Vysokoúrovňové DSL (doménové jazyky) promítáme do nízkoúrovňových strojových instrukcí prostřednictvím mnoha vrstev abstrakcí. Zde je důležité reprezentovat úkol co nejúčinnějším způsobem pro zpracování na úrovni stroje, nikoli fanaticky následovat jedno paradigma. Nejúčinnější je ten, který s menším počtem vrstev a závislostí, nejlépe čitelný, udržovatelný a upravitelný pro člověka, který zajišťuje spolehlivost kódu a testovatelnost, rozšiřitelnost, opětovnou použitelnost, jasnost a flexibilitu konstrukcí metadat na každé úrovni. Věříme, že takový přístup nám umožní získat jak rychlé první výsledky ve vývoji, tak i neztrácet výkon při velkém toku změn ve fázích, kdy projekt již dosáhl vysoké zralosti a složitosti. Pokusíme se zvážit techniky a principy programování z různých paradigmat prizmatem metaprogramování a změnit tím, ne-li samotné softwarové inženýrství, ale alespoň rozšířit jeho chápání novými generacemi inženýrů.
```
Comment (tshemsedinov) [inline content/Abstract.cz.md:3]:
```suggestion
Všechny programy jsou data. Některá data jsou interpretována jako hodnoty, jiná jako typy těchto hodnot a další jako instrukce pro zpracování prvních dvou. Jakákoli paradigmata a programovací techniky jsou jen způsobem, jak vytvořit metadata, která dávají pravidla a posloupnost toku zpracování jiných dat. Multiparadigmatické programování bere to nejlepší ze všech paradigmat a sestavuje z něj syntaktické konstrukce, které umožňují popsat předmětnou oblast jasněji a pohodlněji. Vysokoúrovňové DSL (doménové jazyky) promítáme do nízkoúrovňových strojových instrukcí prostřednictvím mnoha vrstev abstrakcí. Zde je důležité reprezentovat úkol co nejúčinnějším způsobem pro zpracování na úrovni stroje, nikoli fanaticky následovat jedno paradigma. Nejúčinnější je ten, který s menším počtem vrstev a závislostí, nejlépe čitelný, udržovatelný a upravitelný pro člověka, který zajišťuje spolehlivost kódu a testovatelnost, rozšiřitelnost, opětovnou použitelnost, jasnost a flexibilitu konstrukcí metadat na každé úrovni. Věříme, že takový přístup nám umožní získat jak rychlé první výsledky ve vývoji, tak i neztrácet výkon při velkém toku změn ve fázích, kdy projekt již dosáhl vysoké zralosti a složitosti. Pokusíme se zvážit techniky a principy různých programovacích paradigmat prizmatem metaprogramování a změnit jestli ne samotné softwarové inženýrství tím, tak aspoň rozšířit pochopení disciplíny novými generacemi profesionálů.
```

**Comment by tshemsedinov** [comment]
@tursky thanks, fixed and landed

**Pull request merged**